src/qgsquick/qgsquickmapsettings.h

```diff
@@ -163,6 +163,12 @@ class QgsQuickMapSettings : public QObject
     //! \copydoc QgsQuickMapSettings::rotation
     void setRotation( double rotation );
 
+    //! \copydoc QgsQuickMapSettings::backgroundColor
+    QColor backgroundColor() const;
```

Comment:
If you add this as a `Q_PROPERTY` with `NOTIFY`, i will also be available in QML.

Reply:
Good, we could make use of that to set a background rectangle to match canvas bg color, that'll beautify panning et cie.